inc/site-health.php

```diff
@@ -928,12 +929,30 @@ function object_cache_tests( $tests ) {
  * @return array
  */
 function test_object_cache() {
-	if ( ! isset( $_ENV['CACHE_HOST'] ) ) {
+	$cache_host = isset( $_ENV['CACHE_HOST'] ) ? $_ENV['CACHE_HOST'] : null;
+	$service_level = _pantheon_get_header( 'Pcontext-Service-Level' );
+	$redis_unavailable = [ 'basic', 'basic_small' ];
+
+	// If we can't find a service level or cache host, bail early.
+	if ( ! $service_level && ! $cache_host ) {
+		return [];
+	}
+
+	/**
+	 * If Redis is unavailable and the service level is basic, we cannot use
+	 * Redis. Bail early, this test is not helpful.
+	 */
+	if ( ! $cache_host && in_array( $service_level, $redis_unavailable, true ) ) {
+		return [];
+	}
```

Comment:
```suggestion
	/**
	 * If the service level is basic, we cannot use
	 * Redis. Bail early, this test is not helpful.
	 */
	if ( in_array( $service_level, $redis_unavailable, true ) ) {
		return [];
	}
```
Why are we checking cache host instead of just skipping for basic service level?